lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+
+class Restaurant
+  def initialize ( messenger, menu)
+    @messenger = messenger
+    @menu = menu
+    @header = { :from => '+441290211165', :to => '+447527994300'}
+  end
+
+  def request_takeaway
+    takeaway = Takeaway.new(self, @menu)
+  end
+
+  def order_takeaway
+    #calculate time in one hours time
+    delivery_time = Time.now + 3600
+    message_body = "Thank you! Your order was placed and will be delivered before #{delivery_time}"
```

Comment:
Metrics/LineLength: Line is too long. [99/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)